src/modules/RA/DataLayer/components/tabs/StyleTab/Style/PiePieceStyleField.js

```diff
@@ -0,0 +1,196 @@
+import React, { useCallback, useEffect, useState } from 'react';
+
+import { makeStyles } from '@material-ui/core/styles';
+import { Field, useField } from 'react-final-form';
+import { useTranslate } from 'react-admin';
+import { FormControlLabel, Paper, Switch, Table, TableBody, TableCell, TableContainer, TableHead, TableRow } from '@material-ui/core';
+import { SortableContainer, SortableElement } from 'react-sortable-hoc';
+import randomColor from 'randomcolor';
+import styles from './styles';
+import ColorPicker from '../../../../../../../components/react-admin/ColorPicker';
+import DragHandle from '../../../DragHandle';
+import { fieldTypes } from '../../../../../DataSource';
```

Comment:
```suggestion
import React, { useCallback, useEffect, useState } from 'react';
import { useTranslate } from 'react-admin';
import { Field, useField } from 'react-final-form';
import { SortableContainer, SortableElement } from 'react-sortable-hoc';
import { FormControlLabel, Paper, Switch, Table, TableBody, TableCell, TableContainer, TableHead, TableRow } from '@material-ui/core';
import { makeStyles } from '@material-ui/core/styles';
import randomColor from 'randomcolor';

import styles from './styles';
import ColorPicker from '../../../../../../../components/react-admin/ColorPicker';
import DragHandle from '../../../DragHandle';
import { fieldTypes } from '../../../../../DataSource';
```

*Just a bit of reodering (main business deps, then design system…)*